setup.py

```diff
@@ -66,9 +66,7 @@
         'beautifulsoup4',
         'beziers>=0.5.0', # Uses new fontTools glyph outline access
         'cmarkgfm',
-        'collidoscope==0.4.1', # 0.4.0 had a bug that failed to detect
-                               # an ïï collision on Nunito Black.
-                               # (see https://github.com/googlefonts/fontbakery/issues/3554)
+        'collidoscope==0.6.3',
```

Comment:
In the setup.py file we either specify just the dependency name with no version restriction, so that any version available can be used. Or we specify ">=" for some version, when there's a minimum version requirement, due to some new feature of the API of the dependency.

In general we should never use "==", because it makes it much more likely to make it impossible to install fontbakery alongside other programs that may have other similarly restrictive dependency rule.

There was an exception to the ban on "==" in the case of collidoscope, because versions after 0.4.1 were not installable in some systems (due to the kurbopy dependency, if I recall correctly). I should have placed a comment describing that reasoning.

If the code in this PR uses features that were not available prior to collidoscope v0.6.3, then, the correct statement here should be `'collidoscope>=0.6.3',  # some comment explaining why older versions can't be used`

Otherwise, we should try to pick the oldest good version of the dependency and place it here with '>=' and a comment (once the kurbopy problem is figured out)..